chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -17,42 +17,57 @@
 # limitations under the License.
 #
 
+ceph_conf = node[:glance][:rbd][:store_ceph_conf]
+admin_keyring = node[:glance][:rbd][:store_admin_keyring]
+
 ceph_env_filter = " AND ceph_config_environment:ceph-config-default"
 ceph_servers = search(:node, "roles:ceph-osd#{ceph_env_filter}") || []
 if ceph_servers.length > 0
   include_recipe "ceph::keyring"
 else
+
```

Comment:
(I don't feel the empty line is needed, so I'd drop it unless you strongly care about it)